lang/java/pom.xml

```diff
@@ -33,6 +33,13 @@
   <url>http://avro.apache.org</url>
   <description>Avro parent Java project</description>
 
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0</url>
```

Comment:
You want to make this <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
You want a text file and not an HTML file.